components/identity-extensions/backend-oauth-authenticator/src/main/java/org/wso2/carbon/identity/authenticator/backend/oauth/internal/OauthAuthenticatorServiceComponent.java

```diff
@@ -20,25 +20,27 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.osgi.service.component.ComponentContext;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
 import org.wso2.carbon.core.services.authentication.CarbonServerAuthenticator;
 import org.wso2.carbon.identity.authenticator.backend.oauth.OauthAuthenticator;
 
 
-/**
- * @scr.component component.name="org.wso2.carbon.identity.authenticator.backend.oauth.OauthAuthenticator" immediate="true"
- */
-@SuppressWarnings("unused")
-public class OauthAuthenticatorServiceComponent {
 
+public class OauthAuthenticatorServiceComponent  implements BundleActivator {
+
+    private ServiceRegistration pipServiceRegRef;
```

Comment:
These formatting issues has been fixed in a recent PR.In a latest PR this class has been replaced.the master branch now contains a new version of OAuthAuthenticatorServiceComponent.java